raiden/transfer/channel.py

```diff
@@ -1382,6 +1535,8 @@ def create_sendexpiredlock(
     if not merkletree:
         return None, None
 
+    nonce = get_next_nonce(sender_end_state)
```

Comment:
this would be better at `events_for_expired_lock` (Which would be better named as `send_expired_lock`)

Reply:
I will change the name of the function. But the other part of your comment is to generate a new nonce and pass it as a parameter? Thats contradicts what we're doing in `create_sendlockedtransfer` so i am not sure i want to do that.